visitor-app/src/shared/components/form/base.js

```diff
@@ -102,6 +102,17 @@ export const SecondaryButton = styled(Button)`
   border-radius: 0;
 `;
 
+export const DisabledButton = styled(Button)`
```

Comment:
Sounds like this button is always going to be disabled, so we should probably use `styled(Button).attrs((props) => ({ disabled: true }))`

But also, do we want a separate component for a button that is disabled, or do we want buttons that can be disabled? e.g. using the `disabled` attribute to change the styling?

Reply:
I had a play with doing this mainly to transition the background, but then discovered that `background: linear-gradient(...` isn't transitionable.

Can move back to this setup if you want.

Reply:
What if instead of transitioning `linear-gradient` you transition `opacity`?

Reply:
Yeah you can do that, then update the z-index on a delay:
https://css-tricks.com/transitioning-gradients/

Seemed a bit over the top for a small feature.

Reply:
Oh I didn't mean using opacity to do a linear-gradient transition, I just meant that the visual indication that the button is disabled would be it going slightly transparent. But if it looks crappy then don't worry about it 🤷🏼‍♂️

Reply:
Yeah, that would not look good. Grey is standard disabled.

Reply:
OK cool. Well given that we don't have a transition right now anyway (since we're rendering a different component), I think let's go back to having button components that can be disabled, it's cleaner.